rebar.config

```diff
@@ -60,10 +60,10 @@
                    {ref, "9ff46af"}}},
 
         {aebytecode, {git, "https://github.com/aeternity/aebytecode.git",
-                     {ref,"2d599d"}}},
+                     {ref,"9041423"}}},
```

Comment:
That's not latest master, but I will bump soon as well. You now did the majority of the work already

Reply:
It is exactly the version that the referenced `aesophia` points to, that is correct I think?!